docs/public-networks/reference/api/index.md

```diff
@@ -2775,6 +2775,194 @@ curl -X POST -H "Content-Type: application/json" --data '{ "query": "{block (num
+      "logsBloom": "0x00000004000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000200000000000000000000080020000000000000200010000000000000000000001000000800000000000000000000000000000000000000000000000000000100100000000000000000000008000000000000000000000000000000002000000000000000000000",
+      "status": "0x1",
+      "to": "0xfd584430cafa2f451b4e2ebcf3986a21fff04350",
+      "transactionHash": "0x4a481e4649da999d92db0585c36cba94c18a33747e95dc235330e6c737c6f975",
+      "transactionIndex": "0x0",
+      "type": "0x0"
+    },
+    {
+      "blockHash": "0x19514ce955c65e4dd2cd41f435a75a46a08535b8fc16bc660f8092b32590b182",
+      "blockNumber": "0x6f55",
+      "contractAddress": null,
+      "cumulativeGasUsed": "0x1de3e",
+      "from": "0x712e3a792c974b3e3dbe41229ad4290791c75a82",
+      "gasUsed": "0x5208",
+      "effectiveGasPrice": "0x9502f907",
+      "logs": [],
+      "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+      "status": "0x1",
+      "to": "0xd42e2b1c14d02f1df5369a9827cb8e6f3f75f338",
+      "transactionHash": "0xefb83b4e3f1c317e8da0f8e2fbb2fe964f34ee184466032aeecac79f20eacaf6",
+      "transactionIndex": "0x1",
+      "type": "0x2"
+    }
+  ]
+}
+```
+
+# curl GraphQL
+
+```bash
+curl -X POST -H "Content-Type: application/json" --data '{ "query": "{block (number : 100) {transactions{hash} timestamp difficulty totalDifficulty gasUsed gasLimit hash nonce ommerCount logsBloom mixHash ommerHash extraData stateRoot receiptsRoot transactionCount transactionsRoot ommers{hash} ommerAt(index : 1){hash} miner{address} account(address: \"0xd42e2b1c14d02f1df5369a9827cb8e6f3f75f338\"){balance} parent{hash} }}"}' http://localhost:8547/graphql
+```
+
+# GraphQL
+
+```text
+{
+  block(number: 100) {
+    transactions {
+      hash
+    }
+    timestamp
+    difficulty
+    totalDifficulty
+    gasUsed
+    gasLimit
+    hash
+    nonce
+    ommerCount
+    logsBloom
+    mixHash
+    ommerHash
+    extraData
+    stateRoot
+    receiptsRoot
+    transactionCount
+    transactionsRoot
+    ommers {
+      hash
+    }
+    ommerAt(index: 1) {
+      hash
+    }
+    miner {
+      address
+    }
+    account(address: "0xd42e2b1c14d02f1df5369a9827cb8e6f3f75f338") {
+      balance
+    }
+    parent {
+      hash
+    }
```

Comment:
This data is relevant to the block rather than the transaction receipt.

I think it's better to base off the example in https://besu-docs-60seh79zu-hyperledger.vercel.app/development/public-networks/reference/api#eth_gettransactionreceipt

e.g.
```
{ 
  block  (hash: "0x0ed9da11f2b4afdf19710402c1891fd23b9e7494a7c38036832d9db1296e846c") {
    transactions {
      block {
        hash 
        logsBloom
      } 
      hash 
      createdContract { 
        address 
      } 
      cumulativeGasUsed
      gas
      gasUsed
      logs{ 
        topics
      } 
      from {
        address
      } 
      to {
        address
      } 
      index
    }
  }
}  
```